client/scss/utilities/_variables.scss

```diff
@@ -252,6 +252,7 @@ $colors: (
   'accessible-grey-1': (#717171, 'secondary'),
   'accessible-grey-2': (#8f8f8f, 'secondary'),
   'keyline-grey': (#e5e5e5, 'secondary'),
+  'charcoal': (#323232, 'secondary'),
   'transparent': (transparent, 'secondary'),
```

Comment:
Need to add this to the palette.config.yml, so it shows up in cardigan (until we sort out a way of sharing the sass vars with the templates)